configuration/authentication-methods/unverified.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Unverified Users"
+sidebarTitle: "Unverified Users"
+description: "Unverifeid user sign-in with email or phone"
+---
+
+<Snippet file="unverified-users-description.mdx" />
+
+## Enabling Unverified Users
+Enabling Unverified Users is easy and does not require any additional code. Simply follow the steps below to enable Unverified Users for your application.
+
+1. Open your app in the [Rownd platform](https://app.rownd.io/applications). Then click on the **Sign-in methods** tab on the left to adjust your flow.
+
+   ![Finding authentication methods is easy; just select "Sign-in methods" from the nav pane.](/images/sign-in-methods-page.png)
+
+2. Scroll down to the bottom of the page and, and press the toggle to next to **Unverified users**.
```

Comment:
Then press **Save**

Reply:
Good point. Also an extra "and," there

Reply:
Actually that whole wording is messed up.